src/ontology/imports/foodon_product_import.owl

```diff
@@ -34131,6 +34131,81 @@ Fishcakes as defined in the Oxford Dictionary of Food and Nutrition are chopped
+        <oboInOwl:hasExactSynonym xml:lang="en">greenhouse tomato</oboInOwl:hasExactSynonym>
+        <rdfs:label xml:lang="en">tomato (whole, raw, greenhouse)</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004269 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004269">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00001250"/>
+        <obo:IAO_0000115 xml:lang="en">An egg of a rohu fish (Labeo rohita).</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <oboInOwl:hasExactSynonym xml:lang="en">Labeo rohita egg</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">fish rohu egg</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">roho labeo egg</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">rohu egg</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">rui egg</oboInOwl:hasExactSynonym>
+        <rdfs:label xml:lang="en">rohu egg (whole)</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004270 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004270">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00001250"/>
+        <obo:IAO_0000115 xml:lang="en">An egg of an ilish fish (Tenualosa ilisha).</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 xml:lang="en">https://en.wikipedia.org/wiki/Ilish</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-03-04T10:28:23Z</dc:date>
+        <oboInOwl:hasExactSynonym xml:lang="en">hilsa egg</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">hilsa herring egg</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">hilsa shad egg</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">ilish egg</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">ilisha egg</oboInOwl:hasExactSynonym>
+        <rdfs:label xml:lang="en">ilish egg (whole)</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004272 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004272">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00001714"/>
+        <obo:IAO_0000115 xml:lang="en">A compound of casein with a metal.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 xml:lang="en">https://www.merriam-webster.com/dictionary/caseinate</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-03-04T10:37:22Z</dc:date>
+        <rdfs:label xml:lang="en">caseinate</rdfs:label>
```

Comment:
I'm wondering whether "caseinate" was requested?  I suspect a better parent is Casein, http://purl.obolibrary.org/obo/FOODON_03420180 because Wikipedia says https://en.wikipedia.org/wiki/Casein "The most common form of casein is sodium caseinate."

Reply:
Yes, the requested term is 'caseinate'. I even this term in the template sheet, which was shown during Genome Tracker's last meeting. 
That's right, sodium caseinate is the most common form of casein, and I think the second common is calcium caseinate which is used to increase the calcium content. 
`Casein`: A protein substance found in milk or in the seeds of leguminous plants.
`Caseinate`: Any of several salts derived from casein via the coagulation of milk protein. 
It looks to me that a caseinate is a form of casein. I think caseinate can be a sub-class of 'casein' (if we want that)? Casein has other forms as well, like casein-based glues, but several foods, creamers, and toppings all contain a variety of caseinates.